WordPress/src/main/java/org/wordpress/android/ui/people/PeopleInviteFragment.java

```diff
@@ -244,14 +244,18 @@ private boolean endsWithDelimiter(String string) {
         }
 
         for (String usernameDelimiter : USERNAME_DELIMITERS) {
-            if (string.endsWith(usernameDelimiter)) return true;
+            if (string.endsWith(usernameDelimiter)){
```

Comment:
Hey @khaykov , not sure if you made this change due to my earlier braces comment but, just wanted to mention that the brace-less single line is acceptable anyway. Notice that in https://source.android.com/source/code-style.html#use-standard-brace-style, the non-acceptable form is the multiline-without-braces. Thanks!